jobs/texastribune_software_engineer_05_2018.html

```diff
@@ -0,0 +1,65 @@
+
+In this role you will build and improve on core newsroom tools and technologies, including a content management system, a membership and donations portal, an elected officials directory, and more. You will work across the newsroom, advising on technology solutions for editorial, audience, art, and business departments. This is not a data journalism or reporting job, but you will routinely interact with both.
+
+You will be joining a five-person team of engineers and product people, using modern tools and frameworks including Django, React, Webpack, and Docker. We are passionate about user experience and performance, and we care deeply about testing, modularity, and maintainability. Each member brings his or her own technical specialties, and we expect you to have one or two (whether you lean front-end, back-end, ops, or design), but as a smaller team we often work across the stack and value the breadth of skills we gain.
+
+### We want to hear from you if:
+
+* You are enthusiastic about our mission and making newsroom workflows better
+* You are curious and willing to learn, adapt and teach
+* You are meticulous and able to set and meet deadlines
+* You are interested in collaborating and communicating complex ideas effectively
+
+### You should have:
+
+* Experience as a front-end or full-stack developer, with fluency in JavaScript, HTML, CSS
+* Experience or interest in back-end code and API design
+* A commitment to modern web standards and a desire to build performant, user-centered experiences
+* The ability to speak to the technical side of ideas and issues in design/editorial meetings, and ask thoughtful questions to better understand user needs
+
+### It would be nice if you have:
+
+* Experience with Python/Django or other back-end languages and frameworks
+* Experience with a modern front-end build system, like Webpack, Grunt, or Gulp
+* Familiarity with tech and tools for code and operations, such as GitHub, AWS, and automated testing/CI
+* Experience with technologies and standards related to news and journalism, such as CMSes, search, email, social media, advertising, or analytics
+
+We know there are great people who won’t check all of these boxes, and we also know you might bring important skills that we haven’t thought of. If that’s you, don’t hesitate to apply and tell us about yourself.
+
+We believe our office is a dynamic and fun place to work, and we're proud of the job we've done helping to make Texas politics transparent and accessible. Your benefits would include a downtown Austin office with transit stipend; cell phone stipend; medical, vision and dental insurance; a modest 401K match; paid time off plus 11 paid holidays; support for professional training and attending industry conferences; and more.
+
+### If you'd like to apply, please send us:
+
+* A cover letter explaining why you’re excited about the job
```

Comment:
It's unusual these days to ask for a cover letter, is that because you are looking for candidates with a particularly high literacy? It might be worth saying that if that's indeed the case.